### PR TITLE
feat: add schema checking to data before upload

### PR DESCRIPTION
**Pull request by mcmcgrath13**
* add a schema defining the data for each of the files
* check the existence and type of each field

### CODE REVIEW

File: scripts/upload-data.js [hunk 1]
```diff
@@ -101,21 +101,171 @@ const main = async () => {
...
+            {
+              name: "cluster_id",
+              type: "number",
+            },
+          ],
+        },
+        {
+          name: "id",
+          type: "number",
+        },
```
Comment (eldu) [inline scripts/upload-data.js:122]: I see this in the geojson, but do we care that this is there? We can keep this! Though i'm just wondering if we need it as a requirement.

**Review by eldu — APPROVED** [review]
This looks really good Mary! Thank you so much for implementing this! I added in a comment !

**Review by s-bessey — APPROVED** [review]
Looks good to me!

**Review by tdivoll — APPROVED** [review]
this makes sense, and the error messages seem informative enough to point people in the direction to fix any type issues.

File: scripts/upload-data.js (hunk 1 above)
Comment (mcmcgrath13) [reply scripts/upload-data.js:122]: I don't know if we care, but it's there and I think is a requirement of geojson itself, so more an extra layer of safety

**Pull request merged**